test/logging.test.js

```diff
@@ -108,91 +108,65 @@ describe('logging', () => {
 
 	describe('LoggingClient', () => {
```

Comment:
Better to view this file with whitespace-only changes hidden, it's not nearly as bad as it seems.